corehq/apps/data_interfaces/utils.py

```diff
@@ -34,3 +35,33 @@ def add_cases_to_case_group(domain, case_group_id, uploaded_data):
         case_group.save()
 
     return response
+
+
+def archive_forms(user, uploaded_data):
+    response = {
+        'errors': [],
+        'success': [],
+    }
+
+    for row in uploaded_data:
+        xform_id = row.get('form_id')
+        try:
+            xform = XFormInstance.get(xform_id)
```

Comment:
i wasn't sure whether to implement domain checking. i thought since the user was a superuser it'd be convenient to be able to just archive a whole bunch of forms without having to make separate sheets for different domains. but if we do plan on not making it a support feature, definitely needs to check domain. i don't think it'll be hard to implement, so i'll go ahead and do that